### DISCUSSION TITLE
Can I use `caveats` for installation instructions?

### DISCUSSION THREAD

**Discussion by helpermethod** [Homebrew/discussions#2820] [Tap maintenance and Homebrew development] [answered]
Some tools, like SDKMAN!, require to source some files in `.bash_profile` or some other dot file. What would be the best way to display instructions for doing this?

It seems I can do this using the `caveats` block but I suspect this is not the right place for this?

**Comment by SMillerDev** [comment, marked as answer]
> What would be the best way to display instructions for doing this?

Your readme, or documentation website. Unless this is only applicable to homebrew. Then it can be in caveats.
